plugins/base/src/main/kotlin/transformers/pages/tags/SinceKotlinTagContentProvider.kt

```diff
@@ -0,0 +1,38 @@
+package org.jetbrains.dokka.base.transformers.pages.tags
+
+import org.jetbrains.dokka.DokkaConfiguration
+import org.jetbrains.dokka.base.translators.documentables.PageContentBuilder.DocumentableContentBuilder
+import org.jetbrains.dokka.model.doc.CustomTagWrapper
+import org.jetbrains.dokka.pages.ContentKind
+import org.jetbrains.dokka.pages.TextStyle
+
+object SinceKotlinTagContentProvider : CustomTagContentProvider {
+
+    private val TAG_NAME = "Since Kotlin"
```

Comment:
Not sure what you mean. You want to move `private val TAG_NAME = "Since Kotlin"` into `CustomTagContentProvider`? Or make the interface accept a tag name instead of / along with `CustomTagWrapper`?

> Or are there cases of the same a tag content for more than one tag name?

Right now, it's not limited - a provider can handle multiple custom tags (in case the generated content is the same, as with `@throws` and `@exception`), and it is provider's responsibility to filter the needed tags.

I thought about adding an additional method into `CustomTagContentProvider`, like
```kotlin
fun isSupported(tagName: String): Boolean
```
then we would filter out providers and call them only if the processed tag is supported. 

But I think this will limit the usage to just named tags, whereas a plugin developer may want to process all custom doctags (like `BlockQuote`) without knowing their names. Right now it's possible

Reply:
> You want to move private val TAG_NAME = "Since Kotlin" into CustomTagContentProvider?

Yes, `val TAG_NAME ` into `CustomTagContentProvider`. It is more convenient and obvious.

> it is provider's responsibility to filter the needed tags.
In this case the filter can be part of the interface.


1) As for me it is not clear what custom tag (tags) we accept.
2) User is responsible for filter the same tags in two different methods. It is easy to make mistake.